webhook/common/common.go

```diff
@@ -89,3 +89,19 @@ func GetLonghornLabelsPatchOp(obj runtime.Object, requiredLabels, removingLabels
 
 	return fmt.Sprintf(`{"op": "replace", "path": "/metadata/labels", "value": %v}`, string(bytes)), nil
 }
+
+func IsRequiredDataEngineEnabled(ds *datastore.DataStore, backendStoreDriver longhorn.BackendStoreDriverType) error {
```

Comment:
`Is` function but did not return bool, so I would suggest renaming it to `ValidateRequiredDataEngineEnabled` or `CheckRequiredDataEngineEnabled`